src/App.js

```diff
@@ -1,23 +1,49 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { useState } from "react";
 
 function App() {
+  const githubapi = "https://api.github.com/users/";
```

Comment:
Generally, we use upper case for independent constant variables and declare them outside of the function component. Can we change it to `GITHUB_API` and declare it outside App function?